corgi/core/models.py

```diff
@@ -277,7 +277,7 @@ class Type(models.TextChoices):
     components: models.Manager["Component"]
 
     class Meta:
-        indexes = (models.Index(fields=("completion_time",)),)
```

Comment:
is this index not needed ?

Reply:
We don't sort by completion_time anyway, and that's why it was added, so I'd say we remove it for now.